src/Mvc/Mvc.TagHelpers/src/LinkTagHelper.cs

```diff
@@ -194,17 +196,25 @@ public class LinkTagHelper : UrlResolutionTagHelper
         [HtmlAttributeName(FallbackTestClassAttributeName)]
         public string FallbackTestClass { get; set; }
 
+        /// <summary>
+        /// The HTML attribute name defined in the stylesheet to use for the fallback test.
+        /// Must be used in conjunction with <see cref="FallbackTestProperty"/> and <see cref="FallbackTestValue"/>,
+        /// and either <see cref="FallbackHref"/> or <see cref="FallbackHrefInclude"/>.
+        /// </summary>
+        [HtmlAttributeName(FallbackTestHtmlAttributeAttributeName)]
+        public string FallbackTestHtmlAttribute { get; set; }
```

Comment:
This needs to be added to the list of valid modes so if you omit `FallbackTestClass` things don't fall over.